### ISSUE TITLE
A11y_Visual Studio Code Services_Download Insiders_Keyboard: The "See SHA-256 Hashes" control is a button but visually it appears as a link.

### ISSUE THREAD

**Issue by Yashi1201** [microsoft/vscode-docs#5546]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win10;#FTP;#DesktopWeb;#A11ySev2;#WCAG3.2.4;#Keyboard;#Chrome;

## Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Windows Version: Win10
Chrome Version: Version 103.0.5060.134(Official build)(64-bit)

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Launch the application Url : https://code.visualstudio.com/
3. Tab till "Download" button.
4. Scroll down the page and hit enter on "Insiders build" link.
5. Tab till button "See SHA-256 Hashes".
6. Observe that "See SHA-256 Hashes" is a button but visually appearing as link.

## Actual:
The "See SHA-256 Hashes" is a button but visually it is appearing as a link.The visual appearance of this contol as a link is not consistent.

## Expected:
The "See SHA-256 Hashes" control is a button and visually should also appears as a button. All buttons should be consistent in appearance visually.

## User Impact:
Keyboard users will get confused with the control appearance as a link and function as a button.

## Attachment
![Button](https://user-images.githubusercontent.com/93374043/183056005-b285c677-17b6-4d86-ba68-b2b6e2a4d524.gif)

**Comment by gregvanl** [comment]
Won't fix as UI is functional and users will quickly understand the displayed table.
Closing as no "Resolved" state for GitHub issues.

**Comment by Yashi1201** [comment]
As per discussion with SME, "If there is a button and it is visually appearing as link control then it is NAB as long as screen reader should read the correct role for it, visually they can appear as anything".
